internal/provider/auth_token_file_test.go

```diff
@@ -49,9 +49,12 @@ func TestAuthLoginTokenFile_Init(t *testing.T) {
 				consts.EnvVarTokenFilename: "/tmp/vault-token",
 			},
 			expectParams: map[string]interface{}{
-				consts.FieldNamespace:        "",
-				consts.FieldUseRootNamespace: false,
-				consts.FieldFilename:         "/tmp/vault-token",
+				// @TODO
+				// add these expected params back once we also read
+				// these from environment variables
+				// consts.FieldNamespace:        "",
```

Comment:
Can uncomment these lines once these params are also being read from environment variables in the provider's configure method

Reply:
Thanks @vinay-gopalan !

 Neither of these are read from env vars. The reason these are no longer being set in the params is because, if we don't provide a "config" in the `raw` field of this test, then we will never hit this line 

https://github.com/hashicorp/terraform-provider-vault/blob/676ef323b1eea211d44a8b278c9054b65377b24a/internal/provider/auth.go#L265

This is because we can't set defaults on the schema anymore so when we try to access the config here

https://github.com/hashicorp/terraform-provider-vault/blob/676ef323b1eea211d44a8b278c9054b65377b24a/internal/provider/auth.go#L261

it is nil.

So in the case where we are only reading configuration from the environment (which is only possible for FieldFilename here), we won't expect FieldNamespace or FieldUseRootNamespace to be set. But that should be ok because the "defaults" that we expect are the zero values for the types.